product_listprice_list_view/models/stock_picking.py

```diff
@@ -0,0 +1,33 @@
+# Copyright 2019 chrono123 & Quartile Limited
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from odoo import models, fields, api
+
+class StockMove(models.Model):
+    _inherit = 'stock.picking'
+
+    #For VCI receipt: Check if "Currency Amount Price Change' (purchase price) changes
+    # with consecutive effect on filter
+    @api.multi
+    def write(self, vals):
+        # Check if the stock.picking is being confirmed
+        if 'date_done' in vals and self.picking_type_id.code == 'incoming':
```

Comment:
There could be multiple picking records pass through the `write`, it will give you error `self.picking_type_id`